core/ledger/kvledger/txmgmt/txmgr/lockbasedtxmgr/collection_val_ext_test.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package lockbasedtxmgr
+
+import (
+	"testing"
+
+	"github.com/hyperledger/fabric/core/ledger"
+	"github.com/hyperledger/fabric/core/ledger/kvledger/txmgmt/version"
+	"github.com/hyperledger/fabric/core/ledger/mock"
+	"github.com/hyperledger/fabric/protos/common"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTransientCollectionValidation(t *testing.T) {
```

Comment:
I think we should remove the word 'transient'. This test is to make sure that no error is returned for unknown collection types.

Reply:
done